recipes/fastembed/recipe.yaml

```diff
@@ -0,0 +1,72 @@
+context:
+  version: 0.5.1
+
+package:
+  name: fastembed
+  version: ${{ version }}
+
+source:
+  - url: https://pypi.org/packages/source/f/fastembed/fastembed-${{ version }}.tar.gz
+    sha256: 93ccdbead9daadde6af37862c53b92fd15fa3115d20bbddb258faa0a5eb36718
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install .
+
+requirements:
+  host:
+    - python
+    - poetry-core
+    - pip
+  run:
+    - python
+    - huggingface_hub <1.0,>=0.20
+    - loguru <0.8.0,>=0.7.2
+    - mmh3 <5.0.0,>=4.1.0
+    - if: match(python, "<3.10")
+      then: numpy >=1.21,<2.1.0
+      else:
+        - if: match(python, ">=3.10,<3.12")
+          then: numpy >=1.21
+          else:
+            - if: match(python, ">=3.12,<3.13")
+              then: numpy >=1.26
+              else: numpy >=2.1.0
+    - if: match(python, "<3.10")
+      then: numpy >=1.21,<2.1.0
+      else:
+        - if: match(python, ">=3.10,<3.12")
+          then: numpy >=1.21
+          else:
+            - if: match(python, ">=3.12,<3.13")
```

Comment:
Can we make these flat? Nested if / else wasn't really envisioned tbh :D

Reply:
Sure!